libs/api/repository/src/lib/gn4/gn4-repository.ts

```diff
@@ -301,6 +274,34 @@ export class Gn4Repository implements RecordsRepositoryInterface {
     )
   }
 
+  importRecordFromExternalFileUrlAsDraft(
```

Comment:
I think this should be called
```suggestion
  duplicateExternalRecord(
```

Here we're not actually importing (which would mean keeping the title and identifier intact), we're copying the record